pkgs/development/libraries/vcg/default.nix

```diff
@@ -14,7 +14,11 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/include
     cp -r vcg wrap $out/include
-    find $out -name \*.h -exec sed -i 's,<eigenlib/,<eigen3/,g' {} \;
+    find $out -name \*.h -exec sed -i"" \
```

Comment:
https://unix.stackexchange.com/questions/36795/find-sed-search-and-replace

"In the BSD implementation of sed the -i flag needs an argument: the extension of a backup file. For example with -i .bak the command would backup file1.txt as file1.txt.bak first before performing the replacement in the original file. Using an empty argument '' means to not use a backup file, which seems to be what you want."